UI/cypress/e2e/postcode.js

```diff
@@ -64,4 +64,23 @@ Then("they redirects to postcode page", () => {
 Then("the privacy link opens privacy page", () => {
     cy.get('[data-testid="privacy-link"]')
         .should('have.attr', 'href', '/privacy')
-}); 
\ No newline at end of file
+});
+
+
+Then("the contact us link opens contact us page", () => {
+    cy.get('[data-testid="contact-us-link"]')
+        .should('have.attr', 'href', '/contact-us')
+});
+
+Then("they will click the contact us link", () => {
```

Comment:
Any page that is linked to from multiple pages should have its javascript code in the `pages.js` step_definition file so that it's common across all feature files. Something line "When they click on the 'contact us' link"